tests/commands/heap.py

```diff
@@ -292,6 +294,7 @@ def setUp(self) -> None:
         self.expected_tcache_bin_size = 0x20 if ARCH == "i686" or is_64b() else 0x18
         return super().setUp()
 
+    @pytest.mark.skipif(ARCH not in ("i686", "x86_64"), reason=f"Skipped for {ARCH}")
```

Comment:
Is this still arch specific? Do you have an arm machine to test on? would be nice if it at least passed locally.

I am setting up a raspberry pi in the next few days for something related, I should be able to test if you can wait.

Reply:
No problem, it can wait but I'm happy to re-test on my devices too